.env

```diff
@@ -2,8 +2,11 @@ PORT=3006
 DB_NAME='127.0.0.1:27017/PriceAndPromo'
 SC_ATTR="PAP-styles"
 CLIENT=pg
-HOST=127.0.0.1
-PG_PORT=5432
-PG_USER=student
-PG_PASSWORD=student
-DATABASE=priceandpromotions
+PGHOST=127.0.0.1
```

Comment:
Should this be .gitignore'd ? :0

Reply:
hey! that's strange. It's always been in my .gitingore. I will check it out.